data/human outfits.txt

```diff
@@ -3065,3 +3065,37 @@ outfit "City-Ship License"
 	category "Special"
 	thumbnail "outfit/city-ship license"
 	description "You look closely at the silhouette of the ship on the license, but you can't make out what model of ship it is. Perhaps it is an older city-ship, one of the first that carried humanity to greatness among the stars, or an intern just picked the first image that came up for 'city-ship'."
+	
+outfit "NDR-114 Android"
+	category "Systems"
+	licenses
+		Paragon
+	cost 80000
+	thumbnail "outfit/android"
+	"required crew" -1
+	"outfit space" -6

```

Comment:
Why not have it instead take up slots in a unique outfit? Have a computer and charging station (green screwdriver) that costs some sort of resource (power, outfit space, heat or inefficiency), but allows you to have 2-4 of these guys per.

Another change I'll be making